react/features/conference/components/native/styles.js

```diff
@@ -3,6 +3,7 @@ import { ColorSchemeRegistry, schemeColor } from '../../../base/color-scheme';
 import { FILMSTRIP_SIZE } from '../../../filmstrip';
 
 export const NAVBAR_GRADIENT_COLORS = [ '#000000FF', '#00000000' ];
+export const INSECURE_ROOM_NAME_LABEL_COLOR = ColorPalette.warning;
```

Comment:
why the constant if it's only used once?

Reply:
It's exported and being used by the expanded label component

Reply:
Ah, my bad!